braph2genesis/graphs/_MultilayerBU.gen.m

```diff
@@ -0,0 +1,433 @@
+ of nodes with within-layer undirected edges either 0 (absence of connection) 
+ or 1 (existence of connection).
+The connectivity matrices are symmetric.
+There are connections between layers connecting the corresponding nodes.
+
+%% ¡props_update!
+
+%%% ¡prop!
+
+NAME (constant, string) is the name of the multilayer binary undirected graph.
+%%%% ¡default!
+'MultilayerBU'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the multilayer binary undirected graph.
+%%%% ¡default!
+'In a multilayer binary undirected (BU) graph, layers have same or different nodes with within-layer undirected edges either 0 (absence of connection) or 1 (existence of connection). The connectivity matrices are symmetric. There are connections between layers connecting the corresponding nodes.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the multilayer binary undirected graph.
+%%% ¡_prop!
+% % % TEMPLATE (parameter, item) is the graph template to set the graph and measure parameters.
+% % % %%%% ¡_settings!
+% % % 'MultilayerBU'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code for the multilayer binary undirected graph.
+%%%% ¡default!
+'MultilayerBU ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the multilayer binary undirected graph.
+%%%% ¡default!
+'MultilayerBU label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the multilayer binary undirected graph.
+%%%% ¡default!
+'MultilayerBU notes'
+
+%%% ¡prop!
+GRAPH_TYPE (constant, scalar) returns the graph type __Graph.MULTILAYER__.
+%%%% ¡default!
+Graph.MULTILAYER
+
+%%% ¡prop!
+CONNECTIVITY_TYPE (query, smatrix) returns the connectivity type __Graph.BINARY__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.BINARY * ones(layernumber);
+
+%%% ¡prop!
+DIRECTIONALITY_TYPE (query, smatrix) returns the directionality type __Graph.UNDIRECTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.UNDIRECTED * ones(layernumber);
+
+%%% ¡prop!
+SELFCONNECTIVITY_TYPE (query, smatrix) returns the self-connectivity type __Graph.NONSELFCONNECTED__ on the diagonal and __Graph.SELFCONNECTED__ off diagonal.
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.SELFCONNECTED * ones(layernumber);
+value(1:layernumber+1:end) = Graph.NONSELFCONNECTED;                
+
+%%% ¡prop!
+NEGATIVITY_TYPE (query, smatrix) returns the negativity type __Graph.NONNEGATIVE__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.NONNEGATIVE * ones(layernumber);
+
+%%% ¡prop!
+A (result, cell) is the cell containing the multilayer binary adjacency matrices of the multilayer binary undirected graph.
```

Comment:
A (result, cell) is the cell containing the binary supra-adjacency matrix of the multilayer binary undirected graph.